site/app/templates/officeHoursQueue/QueueFilter.twig

```diff
@@ -29,45 +29,24 @@
     <button id="change_code_btn" type="submit" class="btn btn-primary">Change Queue Access Code</button>
   </form>
   <br>
-  <span class="option-title">Filter visible queues</span>
+  <span class="option-title">Modify queue status</span>
   <table class="table table-striped" style="width:100%;">
     <thead>
       <tr>
-        <th scope="col">Filter</th>
-        <th scope="col">Queue Name/Access Code</th>
-        <th scope="col">Open/Close</th>
+        <th style="text-align: left;" scope="col">Queue Name/Access Code</th>
+        <th scope="col">Open?</th>
         <th scope="col">Empty</th>
         <th scope="col">Delete</th>
       </tr>
     </thead>
     <tbody>
       {% for queue in viewer.getAllQueues() %}
-        <tr class="row_color_{{viewer.getIndexFromCode(queue['code'])}}">
-          <td>
-            <input
-              type="checkbox"
-              title="Toggle filter for: {{queue['code']|upper}}"
-              aria-label="Toggle filter for: {{queue['code']|upper}}"
-              id="queue_filter_{{viewer.cleanForId(queue['code'])}}"
-              class="page_loading queue_filter"
-              onchange="updateQueueFilter('{{viewer.cleanForId(queue['code'])}}')"/>
-          </td>
+          <tr>
           <td style="text-align: left;">
-            <label for="queue_filter_{{viewer.cleanForId(queue['code'])}}">
               <b>Name:</b>{{queue['code']}}<br>
               <b>Access Code:</b>{{queue['token']}}</td>
-          </lable>
           <td>
-            <form method="post" action="{{base_url}}/{{queue['code']}}/toggle">
-              <input type="hidden" name="csrf_token" value="{{ csrf_token }}"/>
-              <input type="hidden" name="queue_code" value="{{queue['code']}}"/>
-              <input type="hidden" name="queue_state" value="{{queue['open']}}"/>
-              {% if queue['open'] == 0 %}
-                <button type="submit" class="btn btn-primary filter_settings_btn open_queue_btn">Open</button>
-              {% else %}
-                <button type="submit" class="btn btn-primary filter_settings_btn close_queue_btn">Close</button>
-              {% endif %}
-            </form>
+              <input type="checkbox" id="toggle_queue_{{loop.index}}" onchange="toggleQueue({{loop.index}}, '{{queue['code']}}')" {% if queue['open'] %} checked {% endif %} />
```

Comment:
CSS Ids and classes should use dashes instead of underscores,
see our style guide here: https://submitty.org/developer/coding_style_guide/css

If there is JS logic attached to this id and its too much work to refactor in this PR we can leave it for now